lib/foreman/logging.rb

```diff
@@ -120,10 +120,11 @@ def blob(message, contents, extra_fields = {})
     private
 
     def load_config(environment, overrides = {})
-      fail "Logging configuration 'config/logging.yaml' not present" unless File.exist?('config/logging.yaml')
+      cofig_file = Rails.root.join('config', 'logging.yaml')
```

Comment:
At least you were consistent
```suggestion
      config_file = Rails.root.join('config', 'logging.yaml')
```

Reply:
:facepalm: Thanks :)